models/transactions.go

```diff
@@ -75,7 +75,7 @@ func (tdb *TransactionDB) Transact(t *Transaction) bool {
 
 	// Rollback transaction on any failures
 	defer func() {
```

Comment:
What is this point of this defer func at all, now? And will `err` even be present in this function?